spec/models/ability_spec.rb

```diff
@@ -198,6 +200,8 @@
       it{ should_not be_able_to(:manage, other_event.difficulty_level) }
       it{ should be_able_to(:manage, event.commercials.first) }
       it{ should_not be_able_to(:manage, other_event.commercials.first) }
+      it{ should be_able_to(:create, event.comment_threads.first) }
```

Comment:
I believe that the create action check succeeds for the wrong reasons. We should discuss this test.